package.json

```diff
@@ -44,25 +44,26 @@
         "uuid": "10.0.0"
     },
     "devDependencies": {
-        "@js-soft/eslint-config-ts": "^1.6.9",
+        "@js-soft/eslint-config-ts": "^1.6.13",
         "@js-soft/license-check": "1.0.9",
         "@js-soft/ts-serval": "2.0.10",
         "@types/chai": "^5.0.0",
         "@types/libsodium-wrappers-sumo": "^0.7.8",
-        "@types/mocha": "^10.0.8",
-        "@types/node": "^22.7.4",
+        "@types/mocha": "^10.0.9",
+        "@types/node": "^22.7.7",
         "@types/uuid": "^10.0.0",
+        "@typescript-eslint/eslint-plugin": "^8.10.0",
```

Comment:
can you remove this please? The correct version of this lib is brought by @js-soft/eslint-config-ts